docs/development.md

```diff
@@ -34,7 +34,11 @@ centos7-devel:
     group: 'devel'
     variables:
       katello_devel_github_username: <REPLACE ME>
-      foreman_installer_options_internal_use_only: "--katello-devel-github-username myname --katello-devel-upstream-remote-name origin --katello-devel-extra-plugins theforeman/foreman_remote_execution --katello-devel-extra-plugins theforeman/foreman_discovery"
+      foreman_installer_options_internal_use_only:
+        - "--katello-devel-github-username <REPLACE ME>"
```

Comment:
Given we override the ```foreman_installer_options_internal_use_only``` variable it looks like ```katello_devel_github_username``` isn't used at all actually. This example can be simplified to get rid of it.